pages/members/Member.js

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import styles from "./members.module.css"
+import Image from "next/image"
+import bannerImgDefault from "../../public/blank_banner.png"
```

Comment:
Capitalize

Reply:
as far as I can see from multiple sources, this is not convention to capitilise all imports. Rather, just components.